docs/guide/faq.rst

```diff
@@ -26,29 +26,125 @@ To make NEURON read a file called ``foo.hoc`` when it starts :
-..
 
-    Installation went smoothly, but every time I bring NEURON up, the interpreter prints this strange message: "jvmdll" not defined in nrn.def JNI_CreateJavaVM returned -1
-    -----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Installation went smoothly, but every time I bring NEURON up, the interpreter prints this strange message: "jvmdll" not defined in nrn.def JNI_CreateJavaVM returned -1
+++++++++++++++++++++++++++++++++++++
+
+You must be running an old version of NEURON. Warnings about Java, such as "Can't create Java VM" or "Info: optional feature is not present" mean that NEURON can't find a Java run-time environment. This is of interest only to individuals who are using Java to develop new tools. NEURON's computational engine, standard GUI library, etc. don't use Java.
 
-    You must be running an old version of NEURON. Warnings about Java, such as "Can't create Java VM" or "Info: optional feature is not present" mean that NEURON can't find a Java run-time environment. This is of interest only to individuals who are using Java to develop new tools. NEURON's computational engine, standard GUI library, etc. don't use Java.
 
 What's the best way to learn how to use NEURON?
 -----------------------------------------------
 First be sure to join `The NEURON Forum <https://www.neuron.yale.edu/phpBB/index.php>`_. 
 If you have time, watch our course :ref:`videos <training_videos>`; the material in each course varies, but they generally include a mix of theory and applications.
 Then :ref:`read these suggestions <how_to_get_started>`.
 
+How do I create a NEURON model?
+-------------------------------
+
+By specifying representations in the computer of the three basic physical components of an actual experiment.
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - **Component**
+     - **Wet lab**
+     - **Compuational Modeling**
+
+   * - Experimental preparation - *What is the biology itself?*
+     -
+       brain slice, tissue culture etc.
+     - 
+        specification of what anatomical and biophysical properties to represent in model
+
+   * - Instrumentation - *How will you stimulate it and record results?*
+     -
+       voltage/current clamp, electrodes, stimulator, recorder etc. 
+     - 
+        computational representations of clamps, electrodes etc. 
+
+   * - Control - *How do you automate the experimental protocol?*
+     -
+       programmable pulse generators etc.
+     - 
+        specification of which variables to monitor and record time step, when to stop, integration method, optimization algorithms
+
+
+The classical approach to using NEURON is to specify all three components by writing a program in ``hoc``, NEURON's programming language. You can do this with any editor you prefer, as long as it can save your code to an ASCII text file. Make sure your ``hoc`` files end with the ``extension .hoc``
```

Comment:
extension is just a regular word... don't monospace that, but do monospace the .hoc